custom/icds_reports/sqldata/exports/beneficiary.py

```diff
@@ -20,7 +20,8 @@ def __init__(self, config=None, loc_level=1, show_test=False, beta=False):
         config.update({
             '5_years': 60,
         })
-        super(BeneficiaryExport, self).__init__(config, loc_level, show_test, beta)
+        super(BeneficiaryExport, self).__init__(config, loc_level, show_test, beta,
```

Comment:
what are your thoughts on using `super().__init__()` instead of manually using the class? I've definitely used the wrong class name previously and only caught it because my linter told me i was wrong

Reply:
good call thanks. still living in the past. https://github.com/dimagi/commcare-hq/pull/25295/commits/e3fc35b42b28988248a516d828777867b8769bcc